stats/stats.go

```diff
@@ -308,6 +308,19 @@ func (e *HistogramData) CalcPercentile(percentile float64) float64 {
 	return e.Max // not reached
 }
 
+func (hd *HistogramData) Histogram() *Histogram {
+	res := NewHistogram(0, 10)
+	res.Counter.Min = hd.Min
+	res.Counter.Max = hd.Max
+
+	for _, bucket := range hd.Data {
+		midpoint := (bucket.Start + bucket.End)/2
+		res.RecordN(midpoint, int(bucket.Count))
```

Comment:
this is a very lazy attempt at reverse engineering the histogram data. ideas for a better way?

Reply:
we could export also missing pieces (offset and divider) in json to make it easy reconstruct the original histogram